src/parser/ast_common.h

```diff
@@ -27,6 +27,7 @@ typedef struct {
 	Vector *properties;		// Array of attributes.
 	AST_GraphEntityType t;	// Type of entity.
 	bool anonymous;			// Entity isn't referenced.
+	bool free;              // Entity should be freed.
```

Comment:
Couldn't see any reference to this new property

Reply:
It's unused, you're right! An idea I was playing with, but it didn't turn out to be necessary - will delete this  line.